lib/matplotlib/text.py

```diff
@@ -1657,17 +1657,41 @@ def set_figure(self, fig):
 
 
 class OffsetFrom(object):
+    '''Callable helper class for working with `Annotation`
```

Comment:
I guess that's new; it's [PEP257](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) which suggests triple double quotes.